gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/translator/GolangTranslator.java

```diff
@@ -160,17 +161,17 @@ protected String getSyntax(final Number o) {
 
         @Override
         protected String getSyntax(final SackFunctions.Barrier o) {
-            return "Barrier." + resolveSymbol(o.toString());
+            return "gremlingo." + resolveSymbol(o.toString());
```

Comment:
Wonder if we should save gremlingo as a static constant called GO_PACKAGE_NAME or something along those lines.

Reply:
That might be better. I'll make that change.